src/testrender/cuda/rend_lib.h

```diff
@@ -6,11 +6,23 @@
 
 #include <OSL/oslconfig.h>
 
-#include <Imath/half.h>
+#if defined(__has_include) && __has_include(<Imath/half.h>)
```

Comment:
I think we dropped support for Imath 2.x in main, so this part shouldn't be necessary (probably just a leftover from a previous merge?)